### PR TITLE
Shopify

### PR DESCRIPTION
**Pull request by tiseysoft**

### CODE REVIEW

**Comment by ppolinaua** [comment]
@tiseysoft typo 'take a look at n8n's [HaloPSA integrations](https://n8n.io/integrations/shopify/) list.'

**Comment by ppolinaua** [comment]
@StarfallProjects ✅

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.shopify.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Shopify
 
-[Shopify](https://www.shopify.com/) is an e-commerce platform that allows users to set up an online store and sell their products.
+The Shopify node allows you to automate work in Shopify, and integrate Shopify with other applications. n8n has built-in support for a wide range of Shopify features, including creating, updating, deleting, and getting order, and products.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.shopify.md:3]:
```suggestion
The Shopify node allows you to automate work in Shopify, and integrate Shopify with other applications. n8n has built-in support for a wide range of Shopify features, including creating, updating, deleting, and getting orders and products. 
```

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.shopify.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 # Shopify
 
-[Shopify](https://www.shopify.com/) is an e-commerce platform that allows users to set up an online store and sell their products.
+The Shopify node allows you to automate work in Shopify, and integrate Shopify with other applications. n8n has built-in support for a wide range of Shopify features, including creating, updating, deleting, and getting order, and products. 
+
+On this page, you'll find a list of operations the Shopify node supports and links to more resources.
 
 !!! note "Credentials"
-    You can find authentication information for this node [here](/integrations/builtin/credentials/shopify/).
+    Refer to [Shopify credentials](https://docs.n8n.io/integrations/builtin/credentials/shopify/) for guidance on setting up authentication.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.shopify.md:8]:
```suggestion
    Refer to [Shopify credentials](/integrations/builtin/credentials/shopify/) for guidance on setting up authentication. 
```

**Pull request merged**